rust-symcrypt/src/lib.rs

```diff
@@ -95,6 +106,30 @@ pub fn symcrypt_random(buff: &mut [u8]) {
     }
 }
 
+/// `NumberFormat` is an enum that contains a friendly representation of endianess
+///
+/// `LSB`: Bytes are ordered from the least significant to the most significant, commonly referred to as "little-endian".
+///
+/// `MSB`: Bytes are ordered from the most significant to the least significant, commonly referred to as "big-endian".
+pub enum NumberFormat {
+    LSB,
+    MSB,
+}
+
+impl NumberFormat {
+    /// Converts `NumberFormat` to the corresponding `SYMCRYPT_NUMBER_FORMAT`
+    fn to_num_format(&self) -> symcrypt_sys::SYMCRYPT_NUMBER_FORMAT {
```

Comment:
>to_num_format [](http://example.com/codeflow?start=7&length=13)

Maybe call this `to_symcrypt_format` or something like that? Seems a little bit confusing for NumberFormat to have a `to_num_format` method.

Reply:
Changed name